.eslintrc.json

```diff
@@ -26,7 +26,8 @@
     "semi": ["error", "always"],
     "comma-dangle": ["error", "always-multiline"],
     "no-invalid-this": 0,
-    "prettier/prettier": 1
+    "prettier/prettier": 1,
+    "no-unused-vars": 1
```

Comment:
I think changing this to a warning can eliminate confusion. So it will only turn red when the import doesn't exist vs. it's just unused. (picture shows before change)

![Screen Shot 2021-01-10 at 1 54 30 PM](https://user-images.githubusercontent.com/39889198/104133358-1bdf2f80-5351-11eb-82a3-1eb75f9733bf.png)